services/kommander-flux/2.0.1/templates/kustomization.yaml

```diff
@@ -25,6 +25,7 @@ resources:
 - rbac.authorization.k8s.io_v1_clusterrolebinding_cluster-reconciler-kommander-flux.yaml
 - rbac.authorization.k8s.io_v1_clusterrolebinding_crd-controller-kommander-flux.yaml
 - v1_namespace_kommander-flux.yaml
+- v1_resourcequota_critical-pods-kommander-flux.yaml
```

Comment:
I guess kustomize apply the resources according to the order of the list here? If so, we probably want to place this right after CRDs before flux deployments. @gracedo @takirala

Reply:
https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/sortoptions/#example-3-default-legacy-sorting
Found this kustomize doc -- iiuc we are using the "default legacy sorting" which applies resources in this order:
```
kind: Kustomization
sortOptions:
  order: legacy
  legacySortOptions:
    orderFirst:
    - Namespace
    - ResourceQuota
    - StorageClass
    - CustomResourceDefinition
    - ServiceAccount
    - PodSecurityPolicy
    - Role
    - ClusterRole
    - RoleBinding
    - ClusterRoleBinding
    - ConfigMap
    - Secret
    - Endpoints
    - Service
    - LimitRange
    - PriorityClass
    - PersistentVolume
    - PersistentVolumeClaim
    - Deployment
    - StatefulSet
    - CronJob
    - PodDisruptionBudget
    orderLast:
    - MutatingWebhookConfiguration
    - ValidatingWebhookConfiguration
```

`ResourceQuota`s are applied second after Namespaces, so I think we should be safe here